examples/uploader/README.md

```diff
@@ -7,38 +7,40 @@ This is an example maintainer tool: It makes it possible to add delegations to
 a remote repository, and then to upload delegated metadata to the repository.
 
 Features:
-   - Initialization (like the client example)
+   - Initialization (much like the [client example](../client/))
    - Claim delegation: this uses "unsafe repository API" in the sense that the
      uploader sends repository unsigned data. This operation can be
      compared to claiming a project name on PyPI.org
    - Add targetfile: Here uploader uses signing keys that were added to the
      delegation in the previous step to create a new version of the delegated
-     metadata
+     metadata. The repository will verify signatures on this metadata.
 
 The used TUF repository can be set with `--url` (default repository is
 "http://127.0.0.1:8001" which is also the default for the repository example).
 In practice the uploader tool is only useful with the repository example.
 
 ### Usage with the repository example
 
-In one terminal, run the repository example and leave it running:
+In one terminal, run the [repository example](../repository/) and leave it running:
 ```console
 examples/repository/repo
 ```
 
 In another terminal, run uploader:
 
 ```console
-# initialize with Trust-On-First-Use
+# Initialize with Trust-On-First-Use
 ./uploader tofu
 
-# Then claim a delegation for yourself:
+# Then claim a delegation for yourself (this also creates and stores a new signing key):
 ./uploader add-delegation myrole
 
-# Then add targets to download:
+# Then add a new downloadable target file to your delegated role[^1]:
```

Comment:
I fear footnotes don't work in code blocks.

Reply:
that does make sense :)

Reply:
Added more tweaks. It's not as pretty but should work